### ISSUE TITLE
Is possible to disable the dot notation on the field option?

### ISSUE THREAD

**Issue by carmolim** [xaksis/vue-good-table#898]
### Issue Type (delete the irrelevant ones) 
- [x] Question

### Specs
What version are you using? v2.21.11

What browser? Chrome, Firefox

### Expected Behavior
Want to display data from an API that has a field that contains a `.` on the key.

### Actual Behavior
The data from the fields that have a `.` wont show on the table because of the dot notation. 

### jsfiddle
[https://jsfiddle.net/xyz3h9fa/](https://jsfiddle.net/xyz3h9fa/)

**Comment by Wolf-Byte** [comment]
Did find a solution to this as I am facing the same issue?

**Comment by carmolim** [comment]
No... In my case I was able to change the API return, but if that wasn't possible I would probably need to change the data structure before pass to the table component, not ideal but should work...
